src/App.css

```diff
@@ -1,5 +1,4 @@
 .main_app_container {
-    background-color: #252423;
     height: 100%;
     width: 100%;
     position: absolute;
```

Comment:
why is `position: absolute` required ?

Reply:
Well, because I am giving height and width as 100%, so without position absolute it will take 100% height of the content only. And if the content is not spread across the screen (of full height), this div would not cover the whole screen as I want.

Reply:
okay. As an alternate, could've used flex: 1 also I believe.
But this too is fine